### PR TITLE
cypress:introducing notebookbar tests

### PR DESCRIPTION
**Pull request by Rash419**
Signed-off-by: Rash419 <rashesh.padia@collabora.com>
Change-Id: I60bcadff6d4602a51a40d02d8cbe445b0a180a64

* Resolves: # 
* Target version: master 

### TODO

- [ ] ...

### Checklist

- [ ] Code is properly formatted
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

File: cypress_test/integration_tests/desktop/writer/top_toolbar_spec.js [hunk 1]
```diff
@@ -22,8 +23,7 @@ describe('Top toolbar tests.', function() {
 	});
 
 	it('Apply highlight color.', function() {
-		cy.get('#tb_editbar_item_backcolor')
-			.click();
+		desktopHelper.actionOnSelector('backColor', (selector) => { cy.get(selector).click(); });
```
Comment (lpranam) [inline cypress_test/integration_tests/desktop/writer/top_toolbar_spec.js:26]: now need to rename this file as now it doesn't contain only top toolbar tests

**Review by lpranam — CHANGES_REQUESTED** [review]
please also update the readme, so we know how to run the notebookbar tests,

File: cypress_test/integration_tests/desktop/writer/top_toolbar_spec.js (hunk 1 above)
Comment (Rash419) [reply cypress_test/integration_tests/desktop/writer/top_toolbar_spec.js:26]: > now need to rename this file as now it doesn't contain only top toolbar tests

it contains all top toolbar functionality only

Comment (lpranam) [reply cypress_test/integration_tests/desktop/writer/top_toolbar_spec.js:26]: but it also tests corresponding notebook bar functionality and top toolbar is classic mode specific.

**Comment by Rash419** [comment]
> please also update the readme, so we know how to run the notebookbar tests,

done and I think we don't need to change the name of the spec.In both case upper portion is known as toolbar

**Pull request merged**